Observer/OfflinePaymentObserver.php

```diff
@@ -6,49 +6,24 @@
 
 class OfflinePaymentObserver implements ObserverInterface
 {
-    /**
```

Comment:
maybe need to clean up this file a bit? (breaklines, code alignment)

Reply:
done @ 81b078e0b967d16b53bfb532673fd627c51c857f